src/constants/pools.ts

```diff
@@ -128,3 +128,18 @@ const doubleEstimatePoolsKovan =
 export const DOUBLE_ESTIMATE_POOLS = doubleEstimatePools
 export const DOUBLE_ESTIMATE_POOLS_MATIC = doubleEstimatePoolsMatic
 export const DOUBLE_ESTIMATE_POOLS_KOVAN = doubleEstimatePoolsKovan
+
+// NOTE: add the pool address and its first deposit tx here for other chain
+export const DEPOSIT_TXHASH: Record<string, string> = {
```

Comment:
@geonellov can we group these by network just to be explicit? ...so something like:

```
export const DEPOSIT_TXHASH: { [chainId in ChainId]?: Record<string, string> } = {
  [ChainId.MAINNET]: {
    '0x64dcbdeb83e39f152b7faf83e5e5673faca0d42a': '0xead0a9717cb13feff2ee58d78786b7b2eac68eafe61809d7bfecf67986222ba8', // xSGD:USDC
    ...
  },
  [ChainId.MATIC]: {
    '0x8123c64d6607412c7ac9e880f12245ef22558b14': '0x1baa65bca924e1a189ae68c0a57768d49f17e661de231566a2236383663f3fc7', // xSGD:USDC
    ...
  },
  ...
}
```